src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,14 @@
+import { NavLink } from "react-router-dom";
+import './NavBar.css'
+
+const NavBar = () => {
+  return (
+    <nav id="navbar">
+        <NavLink to="/login">Login</NavLink>
+        <NavLink to="/events">Events</NavLink>
```

Comment:
'/events' route does not exist it must '/'

```suggestion
        <NavLink to="/">Events</NavLink>
```